tex/latex/biblatex/lbx/french.lbx

```diff
@@ -1,7 +1,20 @@
 \ProvidesFile{french.lbx}
 [\abx@lbxid]
 
-% Translations by non-native speaker: this file needs an overhaul!
+\DefineBibliographyStrings{french}{%
```

Comment:
Is this change intended?

Reply:
No, in fact it’s a part of the content of my version of the traduction (for the new package), that were in my clipboard… I don’t know why, but some days ago my computer had a weird trip and has assign the middle-button of the mouse to the "paste" function… So sometimes, when I scroll, it pastes things anywhere.

Reply:
Ah. Thanks.

Reply:
I’ve corrected it in my new commit.